### DISCUSSION TITLE
Benchmark RavenDB vs …

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#15360] [General]
I found these numbers on twitter and I'm curious if you have done a similar test with RavenDB?

Updating the value of a column for 9.4 million rows. How long does it take?

MySQL: 3 minutes
SingleStore: 5.9 seconds

MySQL
- $730/month
- 250 GB Disk
- High Availability
- 4 vCPU
- 32 GB RAM

SingleStore
- $500/month
- 250 GB Disk
- High Availability
- 2 vCPU
- 16 GB RAM

**Comment by ayende** [comment]
Way too many variables to account for a good answer.
Looking at upsert rates, we can do 50K - 100K/ sec without doing anything special and on commodity hardware. 
That means 1 - 3 minutes, we can push that higher with a proper setup, so likely under a minute. 

The problem with a benchmark like that it that it is a pretty meaningless. SingleStore has a column store, which basically means that it can do an operation like that by a single operation.

**Comment by garayx** [comment]
fyi you can see the [Couchbase vs RavenDB](https://ravendb.net/whitepapers/couchbase-vs-ravendb-performance-at-rakuten-kobo) whitepaper here:
https://ravendb.net/whitepapers/couchbase-vs-ravendb-performance-at-rakuten-kobo
